test/Transport/SendmailTest.php

```diff
@@ -27,6 +27,7 @@ class SendmailTest extends TestCase
     public $message;
     public $additional_headers;
     public $additional_parameters;
+    public $operating_system = '';
```

Comment:
Looks like we don't need the default value here.

Reply:
Agreed